apps/pg_subscriptor/lib/handler.ex

```diff
@@ -1,8 +1,9 @@
-defmodule PgHandler do
+defmodule PgSubscriber.Handler do
```

Comment:
btw, from what I understand, the module path should correspond with file path from `lib`/`core`.
So handler should be under `pg_subscriber` dir

Reply:
I think I understand now :smile:  I confused app name with the folder in lib.... hmm let me think.